pages/header.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from constants.header import HeaderConsts
+from pages.base_page import BasePage
+from pages.utils import wait_until_ok
+
+
+class Header(BasePage):
+    log = logging.getLogger("[CreatePostPage]")
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.constants = HeaderConsts()
+
+    """SIGN IN"""
+
+    def sign_in_and_fail(self, login, password):
```

Comment:
I think it's actually two different headers. On the StartPage and on any other page. You cannot do SignIn from HelloPage. So I think this parts should be separated.